back/src/openapi/routes/categories.ts

```diff
@@ -0,0 +1,29 @@
+import { registry } from '../registry'
+import { z } from '../zod'
+
+
+const pathRoot = '/api/v1/categories'
```

Comment:
Esto debería estar en constantes y compartirse entre openapi y estas rutas

Reply:
/api/v1 has be stored as constant in appConfig, and now all routes and openapi use it from there.